theme-switcher/switcher.js

```diff
@@ -0,0 +1,21 @@
+var themeList = document.getElementById('theme-list');
+
+function enableTheme(e) {
+  browser.management.setEnabled(e.target.value, true);
+  e.preventDefault();
+  window.close();
+}
+
+browser.management.getAll().then((themes) => {
+  for (let theme of themes) {
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1289336 is about extending the API to return all extensions.